hangman/hangman.py

```diff
@@ -1,14 +1,19 @@
 import random
+import re
+import math
+
+
 def get_target_word():
     """Selects a random string(target word) from a file"""
     my_list = []
     with open("hangman/data/words.txt", "r") as words:
         for paragraphs in words:
             my_list.append(paragraphs.strip("\n"))
     random_word = str(random.choice(my_list).upper())
-    print(random_word)
+    # print(random_word)
```

Comment:
Please leave this printed for testing purposes. You'll take it out at the end.

Reply:
ok